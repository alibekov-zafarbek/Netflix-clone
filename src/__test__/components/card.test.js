// import React from 'react';
// import { render, fireEvent } from '@testing-library/react';
// import { Card, Player } from '../../components';

// const category = 'series';
// const slideRows = [
//   {
//     title: 'Documentaries',
//     data: [
//       {
//         genre: 'documentaries',
//         maturity: '18',
//         slug: 'tiger-king',
//         description: 'Tiger King description',
//         id: '07adcf19-517f-41b4-8f8b-04fee694bce1',
//         title: 'Tiger King',
//         docId: 'IcRxJBanzE7nmIp2GMGb',
//       },
//     ],
//   },
//   {
//     title: 'Feel Good',
//     data: [
//       {
//         title: 'Juno',
//         genre: 'feel-good',
//         description: 'Juno description',
//         maturity: '0',
//         id: 'a938b5a6-fe25-4a06-8050-353bc7146ccd',
//         slug: 'juno',
//         docId: '4JDgdf5vE0K3YrW9ZnbP',
//       },
//     ],
//   },
// ];