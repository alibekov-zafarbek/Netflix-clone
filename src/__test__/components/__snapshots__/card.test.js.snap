// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> render the <Card /> and toggles  the card feature 1`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Card /> render the <Card /> and toggles 1`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Card /> render the <Card /> with populated data 1`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Card /> render the <Card /> with populated data 2`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Card /> renders the <Card /> with and clicks the card feature 1`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
        data-testid="tiger-king-item-feature"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
        data-testid="juno-item-feature"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Card /> renders the <Card /> with populated data 1`] = `
<div
  class="card__Group-sc-b8c607-2 hxwTgM"
>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Documentaries
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/documentaries/tiger-king/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Tiger King
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Tiger King description
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="card__Container-sc-b8c607-1 dBvHBz"
  >
    <p
      class="card__Title-sc-b8c607-0 kfOkJL"
    >
      Feel Good
    </p>
    <div
      class="card__Enteties-sc-b8c607-5 lkOMeU"
    >
      <div
        class="card__Item-sc-b8c607-8 dCbJxK"
      >
        <img
          class="card__Image-sc-b8c607-7 jYfQgA"
          src="/images/series/feel-good/juno/small.jpg"
        />
        <div
          class="card__Meta-sc-b8c607-6 VXapM"
        >
          <p
            class="card__SubTitle-sc-b8c607-3 ehavFX"
          >
            Juno
          </p>
          <p
            class="card__Text-sc-b8c607-4 cmmkOO"
          >
            Juno description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
